package.json

```diff
@@ -8,10 +8,33 @@
     ".": {
       "require": "./dist/module.cjs",
       "import": "./dist/module.mjs"
+    },
+    "./config": {
+      "require": "./dist/config/index.cjs",
+      "import": "./dist/config/index.mjs",
+      "types": "./dist/config/index.d.ts"
     }
   },
   "main": "./dist/module.cjs",
   "types": "./dist/types.d.ts",
+  "unbuild": {
+    "entries": [
```

Comment:
this looks a little hacky as `nuxt-module-builder` has cjs output off, since it provides its own static stub for module.cjs

this could be moved to `build.config.ts` but it may be confusing as it's not clear this merges with the preset from the builder

Reply:
Any feedback here @danielroe ?